app/views/lead_providers/guidance/npq_usage.html.erb

```diff
@@ -85,7 +78,74 @@
 </p>
 
 
-<h2 id="declaring-that-an-npq-participant-has-started-their-induction" class="govuk-heading-l">
+<h2 id="make-a-declaration" class="govuk-heading-l">Making a declaration for an NPQ participant</h2>
+
+<h3 class="govuk-heading-m">
+  What is a declaration?
+</h3>
```

Comment:
Probably not for this PR, but I wonder how similar the two documentation files are, and whether it would be a good idea to commonise them...

Reply:
They're both almost identical. The only difference is the description of a participant. We've just separated them since there isn't a good home for the overview. Home seems to be about the API in particular and there isn't a general usage guide.